PWGHF/TableProducer/treeCreatorD0ToKPi.cxx

```diff
@@ -61,6 +62,7 @@ DECLARE_SOA_COLUMN(DecayLengthNormalised, decayLengthNormalised, float);
 DECLARE_SOA_COLUMN(DecayLengthXYNormalised, decayLengthXYNormalised, float);
 DECLARE_SOA_COLUMN(Cpa, cpa, float);
 DECLARE_SOA_COLUMN(CpaXY, cpaXY, float);
+DECLARE_SOA_COLUMN(MaxNormalisedDeltaIP, maxNormalisedDeltaIP, float);
```

Comment:
Do you not want to store this in the full table?

Reply:
I added it in the "lite" version because I would like to use it, but I was unsure whether this was the case also for those who use the "full" version. However I added it now (in the end it is only one more variable indeed)